python/lsst/meas/algorithms/matcherSourceSelector.py

```diff
@@ -162,4 +177,38 @@ def _isUsable(self, source):
             and not source.get(self.fluxFlagKey) \
             and self._goodSN(source)
 
+    def _isUsable_vector(self, sourceCat):
```

Comment:
Error: this is a second copy of the method!

Similarly _isUsable appears in two places.

The two implementations seem to differ slightly and I don't know which is correct.

Please always always lint your Python with flake8 before saving (or, better yet, while editing). It will show errors like this. Aside from these two duplicated methods the remaining violations are minor, though still worth fixing.